packages/frontend-vitals-monitoring/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@govuk-one-login/frontend-vital-signs",
+  "version": "0.0.1",
+  "description": "",
+  "type": "module",
+  "esnext": "src/index.ts",
+  "main": "cjs/index.cjs",
+  "module": "esm/index.js",
+  "scripts": {
+    "build": "rollup -c",
+    "test": "jest -c"
+  },
+  "author": "Nick Heal",
+  "license": "ISC",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/govuk-one-login/frontend-vital-signs.git"
+  },
+  "devDependencies": {
+    "@babel/core": "7.24.4",
+    "@babel/preset-env": "7.24.4",
+    "@types/aws-lambda": "^8.10.137",
+    "@types/express": "^4.17.21",
+    "@types/jest": "29.5.7",
+    "@types/node": "^20.12.7",
+    "babel-jest": "29.7.0",
+    "express": "^4.19.2",
+    "jest": "29.7.0",
+    "rollup": "4.14.3",
+    "rollup-plugin-typescript2": "0.36.0",
+    "ts-jest": "29.1.2",
+    "ts-node": "10.9.2",
+    "tslib": "2.6.2",
+    "typescript": "5.4.5"
+  },
+  "types": "./esm/index.d.ts",
+  "dependencies": {
+    "aws-lambda": "^1.0.7",
```

Comment:
I think `aws-lambda` and `forwarded-parse` can probably be deleted here?

Anything under `dependencies` will get installed by the consuming repository so if we can keep this down to just the ones we're using it will help them in managing their dependency trees.

`devDependencies` I'm not so worried about because that's just installed for us doing the development (although it's still good to keep it down to just the ones we're using)

Reply:
Confirmed as resolved